Pictures/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+
+function(preparing_benchmark arg0)
+    add_custom_target(make-directory-benchmark ALL
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_BINARY_DIR}/bin/Benchmark)
+
+    set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin/Benchmark)
+
+    file(COPY ${IMAGES_DIR}/BITMAPA.png
+        DESTINATION ${CMAKE_BINARY_DIR}/bin/Benchmark/Pictures/)
+endfunction()
+
+function(preparing_singlePoint_tests arg0)
+    add_custom_target(make-directory-tests ALL
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${OUTPUT_IMG_PATH}/test_library)
+
+    set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin/UnitTests/TestSinglePointTransformLibrary)
+
+    file(COPY ${IMAGES_DIR}/BITMAPA.png
+        DESTINATION ${CMAKE_BINARY_DIR}/bin/UnitTests/Pictures/)
+endfunction()
+
+function(preparing_QPolygonTest_tests arg0)
+    set(OUTPUT_IMG_PATH ${CMAKE_BINARY_DIR}/bin/UnitTests/TestSinglePointTransformLibrary/GeneratedPicture)
```

Comment:
I see there is misunderstanding how to handle common resources or even not common resources, but handled by common function. Please remind me to discuss this topic on our meeting.